doc/modules/model_evaluation.rst

```diff
@@ -354,6 +356,54 @@ In the multilabel case with binary label indicators: ::
     for an example of accuracy score usage using permutations of
     the dataset.
 
+.. _balanced_accuracy_score:
+
+Balanced accuracy score
```

Comment:
Not sure what correct ordering is here. The current order seems to be approximately alphabetical, but only approximately...